.travis.yml

```
@@ -27,18 +28,16 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
   # Create conda test environment
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy pandas xarray requests beautifulsoup4 lxml netCDF4 pytest-cov pytest-ordering future
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy pandas xarray requests dask toolz netCDF4 portalocker pytest-cov pytest-ordering
   - conda activate test-environment
```

Here's how this is handled in the core code:
```suggestion
  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
  - conda activate test-environment
  - while read requirement; do conda install --yes $requirement; done < requirements.txt
  
```

---

My only concern here is that since we're using develop-3 pysat, it will be installed twice and with conflicting versions.  Eventually this problem will go away, though...

---

true